handlers/handler_instance.go

```diff
@@ -20,7 +20,7 @@ func NewHandlerInstance(
 		config: handlerConfig,
 	}
 
-	h.task = handlerConfig.Queue.NewTask(&taskq.TaskOptions{
+	h.task = handlerConfig.Queue.RegisterTask(&taskq.TaskOptions{
```

Comment:
@nveenjain Any help here? I updated it according to new methods, but it seems docs are quite less descriptive.

Reply:
looking into it!

Reply:
Actually the architecture has changed in v3 @kushthedude, Now what RegisterTask method is present in the `taskq` module itself.
These tasks should be added separately to the queue. (A separate package for this is recommended)
and the queue can be started there... (In worker package/process is recommended)
As mentioned in https://github.com/vmihailenco/taskq#quickstart
detailed example in :- https://github.com/vmihailenco/taskq/tree/master/examples/api_worker